packages/bare-example-notice/package.json

```diff
@@ -11,7 +11,8 @@
   },
   "dependencies": {
     "react": "18.3.1",
-    "react-native": "0.76.7"
+    "react-native": "0.76.7",
+    "react-native-legal": "*"
```

Comment:
If we add local dependency here, then let's remove it from  `packages/bare-example-notice/react-native.config.js` (remove the whole file) and from the `packages/bare-example-notice/tsconfig.json` (the compilerOptions object)